### PR TITLE
RHMAP-17280

### PR DESCRIPTION
**Pull request by aliok**
Document digger-installer variables

### CODE REVIEW

File: docs/digger/installation/installer-variables.asciidoc [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+|What to name the rvm installation script on the node.
+|install-rvm.sh
+
+|rvm_gpg_url
+|The URL of the rvm prerequisite gpg key.
+|https://rvm.io/mpapis.asc
+
+|rvm_gpg_file_name
+|What to name the rvm gpg file on the node.
+|mpapis.asc
```
Comment (PhilipGough) [inline docs/digger/installation/installer-variables.asciidoc]: All rvm vars above can be removed as they will be dropped with this pr https://github.com/aerogear/digger-installer/pull/81

File: docs/digger/installation/installer-variables.asciidoc [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+|The version of Ruby to install on the node.
+|ruby-2.3.3-latest
+
+|node_versions
+|A list of Node versions to install.
+|6
+
+|xcode_install_version
+|The version of the xcode-install tool to install on the node.
+|2.2.0
```
Comment (PhilipGough) [inline docs/digger/installation/installer-variables.asciidoc]: Should now be 2.2.1

File: docs/digger/installation/installer-variables.asciidoc [hunk 3]
```diff
@@ -0,0 +1,295 @@
+== Variables
+
```
Comment (finp) [inline docs/digger/installation/installer-variables.asciidoc]: @wei-lee do we expect users to edit these values? if so, how? (inventory file?)

File: docs/digger/installation/installer-variables.asciidoc [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+|nvm_install_url
+|The URL of nvm installation script.
+|https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh
+
+|nvm_install_file_name
+|What to name the nvm installation script on the node.
+|install-nvm.sh
+
+|ruby_version
```
Comment (matskiv) [inline docs/digger/installation/installer-variables.asciidoc]: We will only use default ruby.
PR: https://github.com/aerogear/digger-installer/pull/81

File: docs/digger/installation/installer-variables.asciidoc [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+|A list of Node versions to install.
+|6
+
+|xcode_install_version
+|The version of the xcode-install tool to install on the node.
+|2.2.1
+
+|gem_packages
+|A list of gems to install on the node. Format: `{ name: <PACKAGE_NAME>, version: <PACKAGE_VERSION> }`
+|Only installs xcode-install by default.
```
Comment (matskiv) [inline docs/digger/installation/installer-variables.asciidoc]: We are adding public_suffix gem to default list.
We could have problem if user overrides this list and doesn't include default ones. @PhilipGough wdyt?

**Comment by matskiv** [comment]
Also we need to document this

**Comment by matskiv** [comment]
Wrong button, sorry.

File: docs/digger/installation/installer-variables.asciidoc (hunk 5 above)
Comment (PhilipGough) [reply docs/digger/installation/installer-variables.asciidoc]: Agreed, these should be forced

File: docs/digger/installation/installer-variables.asciidoc (hunk 4 above)
Comment (PhilipGough) [reply docs/digger/installation/installer-variables.asciidoc]: Correct this is now invalid

Comment (laurafitzgerald) [reply docs/digger/installation/installer-variables.asciidoc]: @matskiv @PhilipGough will remove this section.

File: docs/digger/installation/installer-variables.asciidoc (hunk 5 above)
Comment (laurafitzgerald) [reply docs/digger/installation/installer-variables.asciidoc]: @matskiv what change is required here? should we remove it from the docs as we don't want it to be overriden?

Comment (matskiv) [reply docs/digger/installation/installer-variables.asciidoc]: @laurafitzgerald yes, I guess removing it is safest option. We can later add different list for user defined gems.

File: docs/digger/installation/installer-variables.asciidoc (hunk 3 above)
Comment (laurafitzgerald) [reply docs/digger/installation/installer-variables.asciidoc]: @finp yes the user can override values from the inventory file. This is outline in the run-digger-installer.asciidoc file and I will point to this file form there.

**Comment by PhilipGough** [comment]
@laurafitzgerald LGTM now

**Pull request merged**